.travis.yml

```diff
@@ -1,10 +1,11 @@
 ---
 language: ruby
+dist: xenial
 services:
   - mysql
 env:
   global:
-    - DATABASE_URL=mysql2://root:@localhost/lobsters_dev
+    - DATABASE_URL=mysql2:///lobsters_test
```

Comment:
@pushcx In response to your comment on IRC, I left this change in the .travis.yml to mirror the authentication mechanism [we use in our ansible playbooks](https://github.com/lobsters/lobsters-ansible/commit/e3c374fd14d9fd0cc8fdf14ecd7dbfa4842b7845)